src/main/java/edu/cornell/kfs/fp/batch/xml/DisbursementVoucherPrePaidTravelOverviewXml.java

```diff
@@ -0,0 +1,87 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.NormalizedStringAdapter;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.cornell.kfs.sys.xmladapters.StringToJavaDateAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "DisbursementVoucherPrePaidTravelOverview", namespace = StringUtils.EMPTY)
```

Comment:
Should this be pre_paid_travel?

Reply:
nope, this just needs to be unique, pre_paid_travel is annotated in DisbursementVoucherDetailXml